### ISSUE TITLE
cli: --master in addition to --grid

### ISSUE THREAD

**Issue by matti** [kontena/kontena#2922]
CLI should support global `--master` to issue commands to specific a `--master` and `--grid` without changing the global current master.  setting `KONTENA_MASTER_URL` is not enough, because then it requires `KONTENA_TOKEN` and `KONTENA_GRID` as well. This option would allow user to use the user token without generating a new token.

or then `--grid` should be taken away.

**Comment by kke** [comment]
This was done in #1911 - more specifically at https://github.com/kontena/kontena/pull/1911/files#diff-44c1ed8cab181c9aef6ab291d4b68ad9R19

**Comment by kke** [comment]
I think you can currently already use `KONTENA_MASTER=master_name` env.

**Comment by matti** [comment]
nope? then it will also start using the other envs

**Comment by kke** [comment]
https://github.com/kontena/kontena/blob/master/cli/lib/kontena/cli/config.rb#L105

```ruby
    load_settings_from_env || load_settings_from_config_file
```

```ruby
      # Craft a regular looking configuration based on ENV variables
      def load_settings_from_env
        return nil unless ENV['KONTENA_URL']
```

```ruby
      # Load configuration from default location ($HOME/.kontena_client.json)
      def load_settings_from_config_file
         ...
        self.current_server = ENV['KONTENA_MASTER'] || settings['current_server']
```

**Comment by matti** [comment]
yes this works, but imho docs are confusing or don't indicate how the env's are exclusive

**Comment by matti** [comment]
--master would be intuitive when --grid is there. I'm serious about removing --grid if --master is not implemented.

**Comment by jakolehm** [comment]
Why not use `--platform` that comes via `cloud` plugin?

**Comment by jakolehm** [comment]
If using the cloud is not an option (local testing etc..) then maybe we should extend `--grid` to accept more specific format:

`--grid <master_name>/<grid_name>`

(I hate adding new global options)
